cmd/genassets/main.go

```diff
@@ -15,33 +15,32 @@ import (
 	"flag"
 	"go/format"
 	"io"
+	"net/http"
 	"os"
 	"path/filepath"
 	"strings"
 	"text/template"
+	"time"
 )
 
 var tpl = template.Must(template.New("assets").Parse(`package auto
 
 import (
-	"bytes"
-	"compress/gzip"
 	"encoding/base64"
-	"io/ioutil"
+)
+
+const (
+	ASSETS_BUILD_DATE = "` + time.Now().UTC().Format(http.TimeFormat) + `"
```

Comment:
Those are presumably copies of identically named Windows constants, but even that is borderline. Go does not use all caps or underscores. `AssetsBuildTime` is fine if it should be exported. https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps